.bazelrc

```diff
@@ -6,6 +6,8 @@ coverage --combined_report=lcov
 test --test_output=errors
 test --test_summary=terse
 test --test_verbose_timeout_warnings
+# Set ICU data directory for tests
+test --test_env=HASTUR_ICU_DATA=external/icu-data/
```

Comment:
This doesn't look like trickery to me. Or at least not less tricky than hard-coding a default path in the actual source file. In the future, it should be loaded from an engine config setting instead of the process env, but even then, you would have some "test config" that gets used only for tests, and so you have the same thing happening anyway.

Reply:
We parse urls in a lot of tests, and having to set up a path before attaching gdb will get old fast. :P I know because that's how it used to be with e.g. our default css, but like I said, just something to think about for now. If we only use the stringprep data, we might be able to embed that? It's from a very static rfc, so I doubt the user will have any value in being able to override it.

Reply:
In a future PR, we'll try to statically-link the data into ICU so that we don't have to mess with this file loading nonsense.